cg/models/rnafusion/rnafusion.py

```diff
@@ -93,3 +100,29 @@ class RnafusionAnalysis(AnalysisModel):
     """
 
     sample_metrics: Dict[str, RnafusionQCMetrics]
+
+
+class RnafusionDeliverables(PipelineDeliverables):
+    """Specification for pipeline deliverables."""
+
+    @staticmethod
+    def get_deliverables_template() -> List[dict]:
```

Comment:
This function seems always to return the same. Consider making it an attribute/property of the class so that you don't have to input a static function to the constructor every time you instantiate this class.

Reply:
this is method has been moved.  I've left this as a staticmethod as I was having issues mocking the property